modules/apps/callbacks/go.mod

```diff
@@ -2,19 +2,31 @@ module github.com/cosmos/ibc-go/modules/apps/callbacks
 
 go 1.21
 
+toolchain go1.21.0
+
 replace github.com/cosmos/ibc-go/v7 => ../../../
 
 replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 
 require (
-	cosmossdk.io/api v0.3.1
+	cosmossdk.io/api v0.7.0
+	cosmossdk.io/client/v2 v2.0.0-20230818115413-c402c51a1508
+	cosmossdk.io/core v0.10.0
 	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.0
 	cosmossdk.io/math v1.1.2
-	github.com/cometbft/cometbft v0.37.2
-	github.com/cometbft/cometbft-db v0.8.0
-	github.com/cosmos/cosmos-sdk v0.47.5
-	github.com/cosmos/gogoproto v1.4.10
-	github.com/cosmos/ibc-go/modules/capability v1.0.0-rc1
+	cosmossdk.io/store v1.0.0-alpha.1.0.20230728080422-54ed7dab3982
+	cosmossdk.io/tools/confix v0.0.0-20230818115413-c402c51a1508
+	cosmossdk.io/x/circuit v0.0.0-20230818115413-c402c51a1508
+	cosmossdk.io/x/evidence v0.0.0-20230818115413-c402c51a1508
+	cosmossdk.io/x/feegrant v0.0.0-20230818115413-c402c51a1508
```

Comment:
For future improvements, I think we could try to remove these deps from modules like callbacks